sass/cd/_cd-icons.scss

```diff
@@ -4,11 +4,11 @@
 
 @font-face {
   font-family: 'unocha-icons';
-  src:  url('../icons/unocha-icons.eot?jb5mn9');
-  src:  url('../icons/unocha-icons.eot?jb5mn9#iefix') format('embedded-opentype'),
-    url('../icons/unocha-icons.ttf?jb5mn9') format('truetype'),
-    url('../icons/unocha-icons.woff?jb5mn9') format('woff'),
-    url('../icons/unocha-icons.svg?jb5mn9#unocha-icons') format('svg');
+  src:  url('../img/icons/unocha-icons.eot?jb5mn9');
+  src:  url('../img/icons/unocha-icons.eot?jb5mn9#iefix') format('embedded-opentype'),
+    url('../img/icons/unocha-icons.ttf?jb5mn9') format('truetype'),
+    url('../img/icons/unocha-icons.woff?jb5mn9') format('woff'),
+    url('../img/icons/unocha-icons.svg?jb5mn9#unocha-icons') format('svg');
```

Comment:
irony, fonts in an image directory 🙃

Reply:
Ha. I thought the same to myself. Not for long though :)